orchestrationSpecs/userConfigSchemas.txt

```diff
@@ -0,0 +1,59 @@
+HTTP_AUTH_BASIC: {
+  username: string
+  password: string
+}
+
+HTTP_AUTH_SIGV4: {
+  region:  string
+  service?: string | "es" // default
+}
```

Comment:
I'd elect to remove this entirely, hidden from the user can't we determine if is AOS or AOSS from the endpoint so the user doesn't have to do this work.